### PR TITLE
Student - Automation - Fix Assignment Reminder tests and new tests for Calendar ToDo Item

### PR DESCRIPTION
**Pull request by ndrsszsz**
- New E2E tests for Calendar ToDo Item
- Fix Assignment Reminder E2E tests

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>This pull request will not generate a release note.</td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

[MBL-17342](https://instructure.atlassian.net/browse/MBL-17342)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.69% | 91.68% | 0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cb7e3e40f49b852f0ae3bb33c324cd8c6bdc6abd
</p>

File: Student/StudentE2ETests/student-e2e-tests.yml
Comment (vargaat) [inline Student/StudentE2ETests/student-e2e-tests.yml:1]: Was this only for testing purposes? If yes we could remove it.

File: TestPlans/AssignmentReminderStudentE2E.xctestplan
Comment (vargaat) [inline TestPlans/AssignmentReminderStudentE2E.xctestplan:1]: Was this only for testing purposes? If yes we could remove it.

**Review by vargaat — APPROVED** [review]

File: Student/StudentE2ETests/student-e2e-tests.yml
Comment (ndrsszsz) [reply Student/StudentE2ETests/student-e2e-tests.yml:1]: Deleted in last commit.

File: TestPlans/AssignmentReminderStudentE2E.xctestplan
Comment (ndrsszsz) [reply TestPlans/AssignmentReminderStudentE2E.xctestplan:1]: Deleted in last commit.

**Review by vargaat — APPROVED** [review]

**Review by balintbartok — APPROVED** [review]

**Pull request merged**